src/routes/github/index.ts

```diff
@@ -19,39 +17,40 @@ const routes = (config) => [
 				payload: gitHubWebhookPayload(),
 			},
 		},
-		handler: async (request, h) => {
-			const { payload, headers } = request;
+		handler: async (request: Request, h: ResponseToolkit) => {
+			const { payload, headers } = request as { payload: any; headers: any };
 			const event = headers['x-github-event'];
 			const {
 				repository: { full_name: repositoryFullName },
 			} = payload;
+
 			const streamlabs = new StreamLabs(
-				{ token: config.STREAMLABS_TOKEN },
+				{ token: config.STREAMLABS_TOKEN || '' },
 				request,
 			);
 			const twitchChat = new TwitchChat({
-				botName: config.TWITCH_BOT_NAME,
-				botToken: config.TWITCH_BOT_TOKEN,
-				channel: config.TWITCH_BOT_CHANNEL,
+				botName: config.TWITCH_BOT_NAME || '',
+				botToken: config.TWITCH_BOT_TOKEN || '',
+				channel: config.TWITCH_BOT_CHANNEL || '',
```

Comment:
I'm not sure if this is the best way to handle if any of the configuration values are null/undefined